server/@types/licenceApiImport/index.d.ts

```diff
@@ -288,6 +288,13 @@ export interface paths {
      */
     post: operations['requestAuditEvents']
   }
+  '/run-remove-ap-conditions-job': {
+    /**
+     * Job to remove AP conditions..
+     * @description Triggers a job that removes AP conditions for all licences that are in PSS period and status equal to 'VARIATION_IN_PROGRESS' or 'VARIATION_SUBMITTED' or 'VARIATION_REJECTED' or 'VARIATION_APPROVED'. Requires ROLE_CVL_ADMIN. Requires ROLE_CVL_ADMIN.
```

Comment:
Very minor and not worth doing a commit for (just for future), looks like the API endpoint docs has a slight typo with `Requires ROLE_CVL_ADMIN. Requires ROLE_CVL_ADMIN.`